### PR TITLE
init and configuration rating service

### PR DESCRIPTION
**Pull request by LEHOANGGLAM**
- Initial and configuration rating service 
I added the config to docker-compose.yml and nginx/templates/default.conf.template
server.port=8089.

Could you please check me if there is any mistake or something else is missing, Thank you very much

- Media Service/Fix warning  jakarta.validation.NoProviderFoundException by add Hibernate Validator Maven dependency
"dependency>
           groupId>org.hibernate.validator</groupId
            artifactId>hibernate-validator</artifactId
            version>8.0.0.Final</version
</dependency"
This transitively pulls in the dependency to the Jakarta Bean Validation API.

- Add api get Rating List (this data added manually)
image

### CODE REVIEW

File: rating/pom.xml [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-oauth2-resource-server</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-validation</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-webflux</artifactId>
```
Comment (thiennn) [inline rating/pom.xml:47]: We might not need spring-boot-starter-webflux.

File: rating/pom.xml [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+			<dependency>
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-dependencies</artifactId>
+				<version>${spring-cloud.version}</version>
+				<type>pom</type>
+				<scope>import</scope>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>
+	<repositories>
```
Comment (thiennn) [inline rating/pom.xml:118]: We might not need this repositories and pluginRepositories block. As we not use pre-release version anymore

File: rating/src/main/java/com/yas/rating/service/ProductService.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.reactive.function.client.WebClient;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import java.net.URI;
+
+@Service
+@Transactional
+public class ProductService {
```
Comment (thiennn) [inline rating/src/main/java/com/yas/rating/service/ProductService.java:14]: This ProductService should be removed

**Review by thiennn — COMMENTED** [review]
Pls add GitHub Actions to build and create docker images
In the pom.xml the <sonar.projectKey>nashtech-garage_yas-cart</sonar.projectKey> should be
<sonar.projectKey>nashtech-garage_yas-rating</sonar.projectKey>

**Comment by LEHOANGGLAM** [comment]
i already deleted epositories and pluginRepositories in pom.xml. Please approve this pull request.

**Comment by LEHOANGGLAM** [comment]
I adready removed  repositories and pluginRepositories and add github action. But fail in sonarcloud code analysis and dont know how to fix this.
image

**Pull request merged**